web/packages/teleport/src/services/userEvent/types.ts

```diff
@@ -76,8 +81,71 @@ export enum IntegrationEnrollKind {
   MachineIDKubernetes = 'INTEGRATION_ENROLL_KIND_MACHINE_ID_KUBERNETES',
   EntraId = 'INTEGRATION_ENROLL_KIND_ENTRA_ID',
   DatadogIncidentManagement = 'INTEGRATION_ENROLL_KIND_DATADOG_INCIDENT_MANAGEMENT',
+  AwsIdentityCenter = 'INTEGRATION_ENROLL_KIND_AWS_IDENTITY_CENTER',
+}
+
+/**
+ * IntegrationEnrollStep defines configurable steps for an integration type.
+ */
+export enum IntegrationEnrollStep {
+  /**
+   * AWSIC steps defined for AWS Idenity Center plugin.
+   * Value matches with proto enums defined in the backend.
+   */
```

Comment:
I think this comment should be placed in a line above.

Reply:
Ah right! https://github.com/gravitational/teleport/pull/51097/commits/0b411a0c15927e9e4ac9cdde7465dc0bd8c23979

Looks like this now:
```
/**
 * IntegrationEnrollStep defines configurable steps for an integration type.
 * Value matches with proto enums defined in the backend.
 */
export enum IntegrationEnrollStep {
  /**
   * AWSIC steps defined for AWS Idenity Center plugin.
   */
  ConnectOidc = 'INTEGRATION_ENROLL_STEP_AWSIC_CONNECT_OIDC',
```